src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -34,6 +34,8 @@ const RichTextEditor: React.FC<RichTextEditorProps> = ({
   const [isFocused, setFocus] = React.useState(false);
   const editor = React.useRef<EditorJS>();
   const editorContainer = React.useRef<HTMLDivElement>();
+  const readOnly = React.useRef(disabled);
```

Comment:
It doesn't look right so please add some comment why it's implemented this way.

Reply:
Refactored the fix and added comments